packages/blocks/src/__internal__/utils/block-range.ts

```diff
@@ -41,13 +50,19 @@ export type BlockRange = {
 };
 
 type ExtendBlockRange = {
-  type: 'Title';
+  type: 'Title' | 'DatabaseTitle' | 'DatabaseRichText';
```

Comment:
@zqran please revert these changes for `ExtendBlockRange` since its current design is not intended for this usage at the moment. Special treatment for database title and database rich text would make the maintenance of selection harder. We can postpone related cursor requirements in first version of database block.

/cc @lawvs any idea for a more generic range structure design?

Reply:
>  Special treatment for database title and database rich text would make the maintenance of selection harder.

Yes, I found it when I implemented it.
It should be supported to achieve undo/redo through shortcuts in the database, right?

Reply:
For the native range, we can include the `virgo` instance within the block range to handle multiple rich-text blocks.

For the block range, we need to wait for the product design.

Reply:
In Virgo, you can convert between `VRange` and browser-native range using `toDomRange` and `toVRange`.

Reply:
> In Virgo, you can convert between `VRange` and browser-native range using `toDomRange` and `toVRange`.

In addition, the `block range` is designed to make consistency between the browser-native range and the block range.

It's different from VRange since the user can select multiple blocks at a time, and VRange can't handle range cross blocks.